packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -677,6 +696,7 @@ class BouncingScrollPhysics extends ScrollPhysics {
         leadingExtent: position.minScrollExtent,
         trailingExtent: position.maxScrollExtent,
         tolerance: tolerance,
+        constantDeceleration: decelerationRate == ScrollDecelerationRate.fast ? 1400 : 0
```

Comment:
All of these comparisons should use switch statements. I know there are only 2 enums now, but the style guide requires switch statements. This is because if we add another value to the enum later, the analyzer will catch all of them so we do not leave out a case.
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-if-chains-or--or--with-enum-values